chef_master/source/_templates/nav-docs.html

```diff
@@ -126,8 +131,44 @@
           },
           {
             "title": "Microsoft Windows Guide",
-            "hasSubItems": false,
-            "url": "/windows.html"
+            "hasSubItems": true,
+            "subItems": [
+              {
+                "title": "Microsoft Azure",
+                "hasSubItems": false,
+                "url": "/azure_portal.html"
+              },
+              {
+                "title": "Chef for Microsoft Windows",
+                "hasSubItems": false,
+                "url": "/windows.html"
+              },
+              {
+                "title": "Chef DK on Windows",
+                "hasSubItems": false,
+                "url": "/dk_windows.html"
+              },
+              {
+                "title": "Install the chef-client on Microsoft Windows",
+                "hasSubItems": false,
+                "url": "/install_windows.html"
+              },
+              {
+                "title": "Azure Powershell_Cmdlets",
+                "hasSubItems": false,
+                "url": "azure_powershell.html"
+              },
+              {
+                "title": "Knife Azure",
```

Comment:
Same as above

Reply:
I don't have a better answer right now. Maybe it will come as I write.